src/libraries/Microsoft.PowerFx.Core/Types/FileType.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+namespace Microsoft.PowerFx.Types
+{
+    public enum FileType
```

Comment:
>FileType [](http://example.com/codeflow?start=16&length=8)

Do we need this distinction?
This gets into tricky scenarios about 
- having clear invariants about each of these.  
- what to do with types that aren't on the list

Reply:
Connectors have the notion of Content-Type and media kind
It's unclear if we need to store them